invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/tombstone.html

```diff
@@ -8,17 +8,46 @@
 
 {%- set title = _("Tombstone") + " | " + config.THEME_SITENAME -%}
 
-{% block message %}
-  <h1><i class="fa fa-flash"></i> {{_('Gone')}}</h1>
+{%- block message %}
+  <h1><i class="fa fa-flash"></i> {{ _('Gone') }}</h1>
   <p>
-  {% trans sitename=config.THEME_SITENAME %}
-    The record  you are trying to access was removed from {{sitename}}. The
+  {%- trans sitename=config.THEME_SITENAME %}
+    The record  you are trying to access was removed from {{ sitename }}. The
     metadata of the record is kept for archival purposes.
-  {% endtrans %}
+  {%- endtrans %}
   </p>
-  {% if record and record.removal_reason %}
-  <p>
-    <strong>{{ _("Reason for removal:") }}</strong> {{record.removal_reason}}
-  </p>
-  {% endif %}
+  <div class="ui segment">
+  {%- if record and record.tombstone and record.tombstone.is_visible %}
+    {%- set removal_reason = record.tombstone.removal_reason.title.en or "None" %}
+    {%- if record.tombstone.removed_by.user == "system" %}
+      {%- set removed_by = "System (automatic)" %}
+    {%- elif record.tombstone.removed_by.user %}
+      {%- set removed_by = "User" %}
+    {%- else %}
+      {%- set removed_by = "Unknown" %}
+    {%- endif %}
+    <p>
```

Comment:
could you provide a screenshot to see how it displays?

Reply:
sure :)